.github/workflows/development-environment.yml

```diff
@@ -5,7 +5,6 @@ on:
     - 'Makefile'
     - '.github/workflows/development-environment.yml'
     - '.envrc'
-    - 'requirements*'
```

Comment:
Testing of Python dependencies will go exclusively to the Python versions workflows.